feature_engine/encoding/__init__.py

```diff
@@ -2,6 +2,7 @@
 The module encoding includes classes to transform categorical variables into numerical.
 """
```

Comment:
We shouldn't change this file unless we add a new transformer. 

The BaseCategoricalTransformer is not intended for users to use. Could you please return this file to the original one?

Reply:
Removed BaseCategorialTransformer from init file.